incubation.md

```diff
@@ -0,0 +1,79 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which new Jupyter Subprojects are created and evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a GitHub repository with a development team that follows the Jupyter governance and contribution model. The goals of the incubation process are:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while complying with corporate contribution restrictions.
+* Contributors can work with the community and gather feedback that will help them develop and refine a clear and concise integration proposal.
```

Comment:
For general community projects, I agree. But for individuals at companies that only have permission to contribute to Jupyter, the incubator is their only way of getting experimental stuff out to the community.